shared/fs/header/index.js

```diff
@@ -0,0 +1,4 @@
+// @flow
+import FolderHeader from './container'
+
+export default FolderHeader
```

Comment:
I think the usual style is to put the unconnected component in `foo/index.js` and the connected component in `foo/container.js`, and have everything external use the connected one

Reply:
Ah, I see, both the unconnected and the connected versions are split platform. I suggest `foo/index.{desktop,native}.js` for the unconnected and `foo/container.{desktop,native}.js` + `foo/container.js.flow` for the connected, then

Reply:
Ah I see. Will do, but would you mind if I address these in a separate PR? Was worried it's gonna cause a lot of conflict when rebasing.